tests/robot-tests/tests/public_api/public_api_major_auto_changes.robot

```diff
@@ -0,0 +1,253 @@
+    user clicks button    Confirm new data set version
+
+    user waits until page finishes loading
+    user waits until modal is not visible    Create a new API data set version
+
+Validate the summary contents inside the 'Latest live version details' table
+    user waits until h3 is visible    Draft version details
+    user checks summary list contains    Version    v1.0    parent=id:live-version-summary
+    user checks summary list contains    Status    Published    parent=id:live-version-summary
+    user checks summary list contains    Release    ${RELEASE_1_NAME}    parent=id:live-version-summary
+    user checks summary list contains    Data set file    ${SUBJECT_1_NAME}    parent=id:live-version-summary
+    user checks summary list contains    Geographic levels    Local authority, National, Regional, School
+    ...    parent=id:live-version-summary
+    user checks summary list contains    Time periods    2020/21 to 2022/23    parent=id:live-version-summary
+    user checks summary list contains    Indicators    Enrolments    parent=id:live-version-summary
+    user checks summary list contains    Indicators    more indicators    parent=id:live-version-summary
+    user checks summary list contains    Filters    Academy type    parent=id:live-version-summary
+    user checks summary list contains    Filters    School type    parent=id:live-version-summary
+    user checks summary list contains    Actions    View live data set (opens in new tab)
+    ...    parent=id:live-version-summary
+
+Validate the summary contents inside the 'draft version details' table
+    user waits until h3 is visible    Draft version details
+    user checks summary list contains    Version    v2.0    parent=id:draft-version-summary    wait=%{WAIT_LONG}
+    user checks summary list contains    Status    Action required    parent=id:draft-version-summary
+    ...    wait=%{WAIT_LONG}
+    user checks summary list contains    Release    ${RELEASE_2_NAME}    parent=id:draft-version-summary
+    user checks summary list contains    Data set file    ${SUBJECT_2_NAME}    parent=id:draft-version-summary
+    user checks summary list contains    Geographic levels    Local authority, National, Regional, School
+    ...    parent=id:draft-version-summary
+    user checks summary list contains    Time periods    2020/21 to 2022/23    parent=id:draft-version-summary
+    user checks summary list contains    Indicators    Enrolments    parent=id:draft-version-summary
+    user checks summary list contains    Indicators    more indicators    parent=id:draft-version-summary
+    user checks summary list contains    Filters    Academy type    parent=id:draft-version-summary
+    user checks summary list does not contain    Filters    School type    parent=id:draft-version-summary
+    user checks summary list contains    Actions    Remove draft version    parent=id:draft-version-summary
```

Comment:
Minor, but do we need the `parent=` declaration? Could be removed as there isn't another selector that would make this ambiguous

Reply:
Yeah I've been getting into named parameters a bit more in Robot test writing recently, as sometimes the order of parameters isn't consistent in our keywords and I wonder if we'd use named params from the outset if we did it from scratch.  Agree there's no ambiguity here though so happy to remove.